packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -110,6 +110,9 @@ $DropDown-item-height: 36px;
       border-color: $ms-color-contrastWhiteDisabled;
       color: $ms-color-contrastWhiteDisabled;
     }
+    &.error {
```

Comment:
Also, is it necessary here? I see the same border color being applied down around line 150.